eng/pipelines/dotnet-framework.yml

```diff
@@ -11,9 +11,8 @@ stages:
 - template: ../common/templates/stages/build-test-publish-repo.yml
   parameters:
     ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-      windowsAmdBuildJobTimeout: 300
-      windowsAmdTestJobTimeout: 120
+      windowsAmdBuildJobTimeout: 150
```

Comment:
This is much too small.  There was a leg on Patch Tuesday that took 190 minutes.  I really, really don't want to see timeouts for official releases so we need to have a sufficient buffer, especially considering the extra time it takes to pull down the new Windows images.  I'd be more comfortable with this set to 240 and we can adjust as we get more data.  There can be a lot of variability in build times and I don't want to be too aggressive when setting timeouts.  Also, some test jobs take about 54 minutes which is too close for comfort compared to the default 60 minutes.  That should be set to 90 minutes.

Reply:
good feedback - changed as suggested.